site/content/en/references/kustomize/kustomization/replacements/_index.md

```diff
@@ -198,10 +198,14 @@ Strings are used for mapping nodes. For sequence nodes, we support two options:
 
 2. Index by key-value pair: `spec.template.spec.containers.[name=nginx].image`
```

Comment:
Please add a sentence here, e.g. `If the key-value pair matches multiple elements in the sequence node, all matching elements will be targetted.`